src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs

```diff
@@ -58,7 +58,7 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
         /// </summary>
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
-            // Hack, Only when GetCultureName returns culture.Name, we use CultureInfoMapper 
+            // Hack, Only when GetCultureName returns culture.Name, we use CultureInfoMapper
```

Comment:
Can we avoid the word hack? Maybe it's just me. More like a special case.

Reply:
I fixed it.